src/Settings.ts

```diff
@@ -59,13 +59,14 @@ export default class EditInNeovimSettingsTab extends PluginSettingTab {
           .onChange(async (value) => {
             this.plugin.settings.listenOn = value;
```

Comment:
Changing the port while neovim is open will likely prevent the user from other files as it will no longer be talking to the same instance